notebooks/name-entity-extraction.ipynb

```diff
@@ -0,0 +1,422 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Real time Name entity recognition on the IPU\n",
+    "\n",
+    "Integration of the Graphcore Intelligence Processing Unit (IPU) and the Hugging Face transformer library means that it only takes a few lines of code to perform complex tasks which require deep learning.\n",
+    "\n",
+    "In this notebook we perform **name entity extraction (NER)**  also known as token classification: we use natural language processing models to classify the words inside the prompt. \n",
+    "\n",
+    "\n",
+    "The ease of use of the `pipeline` interface lets us quickly experiment with the pre-trained models and identify which one will work best.\n",
+    "This simple interface means that it is extremely easy to access the fast inference performance of the IPU on your application.\n",
```

Comment:
We advise avoiding things like "[extremely] easy" because if the user has problems (whether our fault or their fault) they will feel even more frustrated if they have been told it will be easy. Maybe something more neutral like "straightforward"